docs/StardustDocs/topics/schemasImportSqlGradle.md

```diff
@@ -0,0 +1,126 @@
+[//]: # (title: Import SQL metadata as a schema in Gradle project)
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Schemas-->
+
+Each SQL database keeps the metadata for all the tables, and 
+this metadata could be used for the schema generation.
+
+**NOTE:** Visit the [page](readSqlDatabases.md) to see how to set up all Gradle dependencies for your project.
+
+### With `@file:ImportDataSchema`
+
+To generate schema for existing SQL table,
+you need to define a few parameters to establish JDBC connection:
+URL, username, and password.
+
+Also, the `tableName` parameter could be specified.
+
+You should also specify the name of the generated Kotlin class 
+as a first parameter of annotation `@file:ImportDataSchema`.
+
+```kotlin
+@file:ImportDataSchema(
+    "ActorSchema",
+    URL,
+    jdbcOptions = JdbcOptions(USER_NAME, PASSWORD, tableName = TABLE_NAME)
+)
+
+import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
+```
+
+```kotlin
+const val URL = "jdbc:mariadb://localhost:3306/imdb"
+
+const val USER_NAME = "root"
+
+const val PASSWORD = "pass"
+
+const val TABLE_NAME = "actors"
+```
+To generate schema for the result of an SQL query,
+you need to define the SQL query itself
+and the same parameters to establish connection with the database.
+
+You should also specify the name of the generated Kotlin class
+as a first parameter of annotation `@file:ImportDataSchema`.
+
+```kotlin
+@file:ImportDataSchema(
+    "TarantinoFilmSchema",
```

Comment:
mention here that file annotation must be above package directive. it's somewhat unexpected and can lead to confusion

Reply:
you can also just write a sample package statement below the annotation, that's also clear